upup/pkg/kutil/kubecfg_builder.go

```diff
@@ -183,20 +184,31 @@ func (c *KubeconfigBuilder) WriteKubecfg() error {
 			return err
 		}
 	}
+	fmt.Printf("Wrote config for %s to %q\n", c.Context, c.KubeconfigPath)
```

Comment:
I don't think I do @kris-nova ... Is this because we should be writing to a stream somewhere?